README.md

```diff
@@ -101,6 +101,7 @@ COMMANDS:
      restart                 Restart processes of your app
      force-https
      sticky-session
```

Comment:
Here, these commands do not have a description on the README, I was wondering why ? @EtienneM

Reply:
Oh I didn't noticed 😅 
I think it's missing. If you can add a description on these commands to that would be nice

Reply:
I'm on it.